templates/travis.yml.jj2

```diff
@@ -26,42 +33,99 @@ python:
 {%block extra_matrix%}
 {%endblock%}
 
+{% block stages %}
 stages:
   - test
+{% if lint_command != False %}
   - lint
+{% endif %}
+{% if moban_command != False %}
+  - moban
+{% endif %}
 
+{% endblock %}
 .disable_global: &disable_global
+  addons: false
+  cache: false
+  env: {}
+  python: false
   before_install: false
-  install: true
+{# Travis-CI documents install: true to skip the install phase, however
```

Comment:
this needs rewording, or even removing.  `install: true` does skip the install phase by executing `/bin/true`, and `install: false` uses the default `install` provided by travis, so `false` restores the default behaviour.

Reply:
https://github.com/moremoban/pypi-mobans/issues/101